opentech/apply/activity/messaging.py

```diff
@@ -368,6 +369,7 @@ class SlackAdapter(AdapterBase):
         MESSAGES.DELETE_SUBMISSION: '{user} has deleted {submission.title}',
         MESSAGES.DELETE_REVIEW: '{user} has deleted {review.author} review for <{link}|{submission.title}>.',
         MESSAGES.CREATED_PROJECT: '{user} has created a Project: <{link}|{project.name}>.',
+        MESSAGES.UPDATE_PROJECT_LEAD: 'The lead of <{link}|{project.name}> has been updated from {old_lead} to {project.lead} by {user}',
```

Comment:
Probably worth differentiating being submission and project in the notifications text for clarity.

`The lead of project <{link}|{project.name}>`?

Reply:
Fixed and rebased.